services/dedup/badger/badger.go

```diff
@@ -77,6 +77,9 @@ func NewBadgerDB(conf *config.Config, stats stats.Stats, path string) *Dedup {
 }
 
 func (d *BadgerDB) Get(key string) (int64, bool, error) {
+	start := time.Now()
+	defer d.stats.NewTaggedStat("dedup_get", stats.TimerType, stats.Tags{"mode": "badger"}).Since(start)
```

Comment:
```suggestion
	defer d.stats.NewTaggedStat("dedup_get_seconds", stats.TimerType, stats.Tags{"mode": "badger"}).Since(start)
```

Reply:
seconds seems weird in a metric, probably I should add time instead?

Reply:
https://docs.chronosphere.io/ingest/metrics-traces/collector/mappings/prometheus/prometheus-recommendations

Reply:
Also prometheus itself suggest such naming convention: https://prometheus.io/docs/practices/naming/

Reply:
Even better name would be `dedup_get_duration_seconds `